test/test_frames.py

```diff
@@ -113,10 +113,14 @@ def test_data_frame_has_correct_flags(self):
             'END_STREAM', 'PADDED'
         ])
 
-    def test_data_frame_serializes_properly(self):
+    @pytest.mark.parametrize('data', [
+        b'testdata',
+        memoryview(b'testdata')
+        ])
```

Comment:
Have to keep it clean. Done.  I need to adjust my flake8 settings. I have too many warnings on right now and that makes me ignore some of them.